src/modules/70-pipeline/pages/inputSet-list/InputSetListView.tsx

```diff
@@ -176,7 +178,7 @@ const RenderColumnMenu: Renderer<CellProps<InputSetLocal>> = ({ row, column }) =
               ;(column as any).goToInputSetDetail?.(data)
               setMenuOpen(false)
             }}
-            disabled={!(column as any).canUpdate}
+            disabled={!(column as any).canUpdate || isPipelineInvalid}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9275140)
```suggestion
            disabled={!(column as unknown).canUpdate || isPipelineInvalid}
```